src/App.tsx

```diff
@@ -158,13 +158,16 @@ const App = () => {
 		'descope-login-container': !isWideContainer
 	});
 
+	const form = { userCode: urlParams.get('user_code') || '' };
```

Comment:
maybe we can generalize this to pass any url param of `form_lorem_ipsum` to ` form: { loremIpsum }`

Reply:
@asafshen we are in a super short timeframe with this Device Flow feature lets focus and do only what is must..
if the current code is ok Yala lets merge.. (or fix ofc if needed)